gateway-service-impl/src/test/java/org/hypertrace/gateway/service/baseline/BaselineServiceImplTest.java

```diff
@@ -229,7 +230,8 @@ private FunctionExpression getFunctionExpressionForAvgRate(
             Expression.newBuilder()
                 .setLiteral(
                     LiteralConstant.newBuilder()
-                        .setValue(Value.newBuilder().setLong(60).setValueType(ValueType.LONG))))
+                        .setValue(
+                            Value.newBuilder().setString("PT1M").setValueType(ValueType.STRING))))
```

Comment:
We should make sure to test this both ways - with an ISO string and a plain long.